findbugs-exclude.xml

```diff
@@ -0,0 +1,5 @@
+<FindBugsFilter>
+     <Match>
+       <Class name="~.+\.Messages" />
```

Comment:
> to be sure to not break anything if the dependency is not present

Should be straightforward enough to have the mojo check the project dependencies for the FindBugs annotations library.